apps/src/sites/studio/pages/projects/index.js

```diff
@@ -21,6 +22,12 @@ $(document).ready(() => {
         showGallery={showGallery}
       />, gallerySwitcher);
 
+    const projectsHeader = document.getElementById('projects-header');
+    ReactDOM.render(
+      <HeadingBanner
+        headingText={i18n.projectGalleryHeader()}
+      />, projectsHeader);
+
```

Comment:
Given that projects-header and gallery-switcher are adjacent in the DOM, could we have them share a single React root node (instead of having two trees)?